### DISCUSSION TITLE
Handle multiple m3u files/links?

### DISCUSSION THREAD

**Discussion by bruor** [trix7777/m2strm#9] [Ideas]
I've got 25 M3U files for VOD content I'd like to use with your tool.  Any chance you could make it possible to feed more than one URL to it and have it download/combine/process them in one shot? 

My plan right now is to maintain 25 different directories and call the tool 25 times with different parameters.

**Comment by trix7777** [comment]
I'm sorry but that will never happen, it's a user scenario this app is not intended for. The "merging" alone is a project by itself.
Thanks for the feedback though =)
